test_settings.py

```diff
@@ -28,6 +28,10 @@ def root(*args):
     }
 }
 
+FEATURES = {
+    'DISABLE_VERIFICATION_EMAIL_NOTIFICATION': False,
+}
+
```

Comment:
Currently, no tests use this test setting. Will you include a test?
Sorry if I missed something.

Reply:
@navinkarkera Thanks for catching that. I missed adding the test for it. Have done so now.